sql/analyzer/rules.go

```diff
@@ -218,6 +219,14 @@ func resolveDatabase(ctx *sql.Context, a *Analyzer, n sql.Node) (sql.Node, error
 	return n, nil
 }
 
+var dualTable = func() sql.Table {
+	t := mem.NewTable("dual", sql.Schema{
+		{Name: "dummy", Source: "dual", Type: sql.Text, Nullable: false},
+	})
+	_ = t.Insert(sql.NewRow("x"))
```

Comment:
Why is necessary insert something in the dual table?

Reply:
`DUAL` is a single-row, single-column table.